### PR TITLE
[zigbee2mqtt] Add chart

### PR DESCRIPTION
**Pull request by ishioni**
#### Special notes for your reviewer:
We have zwave, time for zigbee
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/zigbee2mqtt/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+$ helm repo add billimek https://billimek.com/billimek-charts/
+$ helm install billimek/zigbee2mqtt
+```
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+helm install --name my-release billimek/zwave2mqtt
```
Comment (billimek) [inline charts/zigbee2mqtt/README.md:17]: Can you please change this to `zigbee2mqtt` ?

File: charts/zigbee2mqtt/values.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: koenkk/zigbee2mqtt
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+timezone: "Europe/Amsterdam"
```
Comment (billimek) [inline charts/zigbee2mqtt/values.yaml:11]: Would you consider defaulting this to `UTC` ?

File: charts/zigbee2mqtt/README.md (hunk 1 above)
Comment (ishioni) [reply charts/zigbee2mqtt/README.md:17]: Oh my, looks like atom didn't save my version of this file. Sorry about that

File: charts/zigbee2mqtt/values.yaml (hunk 2 above)
Comment (ishioni) [reply charts/zigbee2mqtt/values.yaml:11]: Sure

**Pull request merged**